### PR TITLE
fix sharedstorage it

### PR DESCRIPTION
**Pull request by J-shang**
1. mount point in local may late to umount when stop experiment in `shared-storage-remote-azureblob`, so keep two shared storage tests away from each other.
2. hide tensorboard not installed error in `nnictl_stderr`

### CODE REVIEW

File: ts/nni_manager/core/nniTensorboardManager.ts [hunk 1]
```diff
@@ -114,9 +114,9 @@ class NNITensorboardManager implements TensorboardManager {
     }
 
     private setTensorboardVersion(): void {
-        let command = `python3 -c 'import tensorboard ; print(tensorboard.__version__)'`;
+        let command = `python3 -c 'import tensorboard ; print(tensorboard.__version__)' 2>&1`;
         if (process.platform === 'win32') {
-            command = `python -c "import tensorboard ; print(tensorboard.__version__)"`;
+            command = `python -c "import tensorboard ; print(tensorboard.__version__)" 2>&1`;
```
Comment (QuanluZhang) [inline ts/nni_manager/core/nniTensorboardManager.ts:119]: 1 means stdout, you redirect stderr message to stdout? nnictl also collect stdout right?

Comment (QuanluZhang) [reply ts/nni_manager/core/nniTensorboardManager.ts:119]: maybe " >/dev/null 2>&1", not sure whether it works in windows...

Comment (J-shang) [reply ts/nni_manager/core/nniTensorboardManager.ts:119]: no, stdout will be collected by nnimanager

Comment (QuanluZhang) [reply ts/nni_manager/core/nniTensorboardManager.ts:119]: ok

**Review by QuanluZhang — APPROVED** [review]

File: ts/nni_manager/core/nniTensorboardManager.ts (hunk 1 above)
Comment (J-shang) [reply ts/nni_manager/core/nniTensorboardManager.ts:119]: tensorboard manager needs the stdout of this command to check the version, so `>/dev/null` will not work?

**Review by ultmaster — APPROVED** [review]

**Pull request merged**